l10n_be_cooperator/reports/tax_shelter_shares_report.xml

```diff
@@ -18,8 +18,9 @@
                 <h2 style="font-size: 3em;">
                     <span>Déclaration Tax Shelter</span>
                     <br />
-                    <small>Attestation concernant une des quatre années suivant
-                        l’acquisition de parts
+                    <small>
+                        Attestation concernant une des quatre années suivant
+                        l'acquisition de parts
                     </small>
```

Comment:
Est-ce que ce n'est pas justement ce qu'on veut éviter? cela va ajouter un `\n' au début de la source string à traduire non?

Reply:
the difference is [here](https://github.com/coopiteasy/vertical-cooperative/pull/380/commits/5087dce3738913808323b6bf0b2708ba2a4d1c33#diff-b511370994b9b1f4376c717dd95456d984c77954ef93c18808248aa2b86b1d86R104). in this case, it’s part of an html block with other tags so it won’t help (and it should be in french if it would need to be translated 😆), but the general rule is:
*   for content in html tags, beginning and ending with a newline is ok because they are trimmed, and it’s better to balance the tags
*   for content in odoo tags, remove any beginnig and ending whitespace